database.go

```diff
@@ -1321,7 +1374,11 @@ func (db *datastore) GetPosts(cfg *config.Config, c *Collection, page int, inclu
 
 	limitStr := ""
 	if page > 0 {
-		limitStr = fmt.Sprintf(" LIMIT %d, %d", start, pagePosts)
+		if db.driverName == driverPostgres {
+			limitStr = fmt.Sprintf(" LIMIT %d OFFSET %d", pagePosts, start)
+		} else {
+			limitStr = fmt.Sprintf(" LIMIT %d, %d", start, pagePosts)
+		}
```

Comment:
I think we can extract these `LIMIT` clauses into a new helper func that works with any dialect, so we don't have to repeat this logic every time.